api/envoy/extensions/upstreams/http/v3/http_protocol_options.proto

```diff
@@ -163,4 +164,22 @@ message HttpProtocolOptions {
   // upstream filters will count as a final response if hedging is configured.
   // [#extension-category: envoy.filters.http.upstream]
   repeated filters.network.http_connection_manager.v3.HttpFilter http_filters = 6;
+
+  // Configuration options for Header Validation (UHV).
+  // UHV is an extensible mechanism for checking validity of HTTP responses.
```

Comment:
The rest of the comments seem to refer to a specific extension (UHV), rather than a generic interface/extension. In a sense, the comments look more like a boolean definition of whether to use the specific extension or not. Can this be more generic?

Reply:
I've tried to clarify in the comment, what happens when there is nothing specified in this field, since this is going to be the most common scenario.